metrics/adapters/tagless.go

```diff
@@ -14,14 +14,19 @@
 
 package adapters
 
-import "github.com/uber/jaeger-lib/metrics"
+import (
+	"time"
+
+	"github.com/uber/jaeger-lib/metrics"
+)
 
 // FactoryWithoutTags creates metrics based on name only, without tags.
 // Suitable for integrating with statsd-like backends that don't support tags.
 type FactoryWithoutTags interface {
 	Counter(name string, help string) metrics.Counter
 	Gauge(name string, help string) metrics.Gauge
-	Timer(name string, help string) metrics.Timer
+	Timer(name string, help string, buckets []time.Duration) metrics.Timer
+	Histogram(name string, help string, buckets []float64) metrics.Histogram
 }
```

Comment:
given these breaking changes to the SPI, why not switch all functions to take an options struct?